Added observations on quotienting by torsion modules

iblech opened the pull request:
This commit adds a discussion of the Serre quotient of the category of modules over some ring by the subcategory of torsion modules, which I'm told is the prototypical example of a Serre quotient.

Please check that the identification of the quotient given in the commit is correct. A result which is so basic must either be wrong or well-known, but I didn't find a statement of this kind in the literature.

If desired, I will gladly improve the wording or improve the pull request in other ways.

aisejohan commented:
Hi! This is great! Now, I want you to change this in the following way (if you have time):
-- Make a new section in examples.tex and put the material there (if we need it later somewhere else, then we'll move it back to an earlier spot --- basically right now I do not know where it goes),
-- In the text in homology.tex put a reference to the section in examples.tex
-- The stacks project does not yet define torsion modules; the safest thing (for now) might be to restrict to the case where the ring A is a domain where we do have a discussion of torsion elements (see http://stacks.math.columbia.edu/tag/0549). Then the total quotient ring becomes the fraction field K which is a field (so easier than the case of a general ring where the total quotient ring need not even be a product of fields).
-- You should discuss two cases leading to two propositions: (all modules)/(torsion modules) = vector spaces over K and when A is a Noetherian (!) the case (finitely generated modules)/(finitely generated torsion modules) = finite dim vector spaces over K. Namely, if A is not Noetherian, then it may not be true that the torsion module of a finitely generated module is finitely generated. (Nor does it hold for coherent modules if A is a coherent ring, I think).
-- Do not assign tags because I prefer to do this myself.

If you have questions we can continue discussing over here or by email.

iblech commented:
Thank you for the swift response and the great suggestions! I believe I incorporated all of them, and in doing so was able to streamline the proof. (Note to bystanders: In addition to being messy, the first version included an error. Without the integrality hypothesis, the total ring of fractions k need to be a field; thus it's not true that any k-module is free. This breaks several arguments.)

Please let me know if you want me to further amend the pull request or squash the commits into a single one.

Also note that I added a short remark to documentation/git-howto to not bother with assigning tags (as per your comment).

aisejohan commented:
OK, I merged it and made a few changes.
